src/Microsoft.Data.Analysis/ArrowStringDataFrameColumn.cs

```diff
@@ -118,11 +139,11 @@ private void SetValidityBit(long index, bool value)
         }
 
         /// <summary>
-        /// Returns an enumerable of immutable buffers representing the underlying values in the Apache Arrow format
+        /// Returns an enumeration of immutable buffers representing the underlying values in the Apache Arrow format
         /// </summary>
-        /// <remarks>Null values are encoded in the buffers returned by GetReadOnlyNullBitmapBuffers in the Apache Arrow format</remarks>
+        /// <remarks><see langword="null" /> values are encoded in the buffers returned by GetReadOnlyNullBitmapBuffers in the Apache Arrow format</remarks>
         /// <remarks>The offsets buffers returned by GetReadOnlyOffsetBuffers can be used to delineate each value</remarks>
-        /// <returns>IEnumerable<ReadOnlyMemory<byte>></returns>
+        /// <returns>An enumeration of <see cref="ReadOnlyMemory{Byte}"/> objects</returns>
```

Comment:
In the Linq docs, we don't use this pattern.

[https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.select?view=netcore-3.0](https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.select?view=netcore-3.0)

```
Returns
IEnumerable<TResult>
An IEnumerable<T> whose elements are the result of invoking the transform function on each element of source.
```

Should we follow these established patterns?

Also, it would be nice to say what these object represent. Like in the link above `whose elements are the result of invoking the transform function on each element of source.`

So:

`An IEnumerable<ReadOnlyMemory<byte>> whose elements are the raw data buffers for the UTF8 string values.`

(obviously escaping the `<` and `>` characters however XML docs need it)

Reply:
Also don't forget the dot at the end of the sentence 😄